components/deck/build-deck.webc

```diff
@@ -226,3 +226,144 @@
   }
 }
 </style>
+
+<style webc:bucket="slides-theme">
+  @layer slide.theme {
+
+    slide-deck {
+
+      /* background colors */
+      --slide-background: hsl(200 12% 95.098%);
+      --slide-callout-color: hsl(193deg 28% 87%);
+      --slide-mark-bg-color: var(--slide-callout-color);
+
+      /* content colors */
+      --slide-text-color: hsl(200deg 12% 4.902%);
+      --slide-text-light-color: hsl(192.86deg 10% 27.451%);
+      --slide-accent-color: hsl(330.2deg 67.742% 42.549%);
+      --slide-border-color: hsl(195deg 10% 50%);
+      --slide-shadow-color: hsl(195deg 30.769% 94.902% / 0.4);
+      --slide-action-color: hsl(195deg 53% 27%);
+      --slide-active-color: hsl(330deg 52.857% 27.451%);
+      --slide-highlight-color: hsl(330deg 52.857% 27.451%);
+      --slide-feature-color: hsl(195.65deg 45.098% 20%);
+      --slide-strikeout-color: var(--slide-highlight-color);
+  
+      /* fonts */
+      --slide-os-serif-font: palatino, 'palatino linotype', 'palatino lt std', 'book antiqua',
+        georgia;
+      --slide-os-sans-font: 'helvetica neue', helvetica, arial;
+      --slide-os-code-font: sfmono-regular, consolas, monaco;
+
+      --slide-serif-font: var(--slide-web-font-serif, var(--slide-os-serif, ui-serif)), serif;
+      --slide-sans-serif-font: var(--slide-web-font-sans, var(--slide-os-sans-font, ui-sans-serif)), sans-serif;
+      --slide-code-font: var(--slide-web-font-code, var(--slide-os-code-font, ui-monospace)), monospace, serif;
+
+      background: var(--slide-background);
+      color: var(--slide-text-color);
+      color-scheme: light;
+    }
+
+    body {
+      font-family: var(--slide-sans-serif-font);
+      font-size: var(--slide-normal-text);
+      font-size-adjust: from-font;
+      line-height: 1.3;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      font-family: var(--slide-sans-serif-font);
+    }
+
+    :focus-visible {
+      outline: medium dotted var(--slide-accent-color);
+      outline-offset: 0.25em;
+    }
+
+    button {
```

Comment:
These button styles are messing with the slide-activation buttons, most noticeable in grid view.

Reply:
Good catch. I cleaned up the button styles in https://github.com/oddbird/eleventy-plugin-slide-deck/pull/2/commits/dc24ca9f459aa92d3fb1f695e74a7a2d71a5c85e. I think that fixes all the button issues?